command/tls_cert_create.go

```diff
@@ -0,0 +1,289 @@
+package command
+
+import (
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
```

Comment:
`ioutil` is being deprecated so we're trying not to import it in new code. Take a look at the documentation of functions you call and replace them with their equivalents from the `os` package please.

Reply:
Will do